packages/@aws-cdk/aws-cloudtrail/README.md

```diff
@@ -31,25 +31,29 @@ For example, to log to CloudWatch Logs
 import cloudtrail = require('@aws-cdk/aws-cloudtrail');
 
 const trail = new cloudtrail.CloudTrail(stack, 'CloudTrail', {
-    sendToCloudWatchLogs: true
+  sendToCloudWatchLogs: true
 });
 ```
 
-This creates the same setup as above - but also logs events to a created CloudWatch Log stream. By default, the created log group has a retention period of 365 Days, but this is also configurable.
+This creates the same setup as above - but also logs events to a created CloudWatch Log stream.
+By default, the created log group has a retention period of 365 Days, but this is also configurable.
 
-
-For using CloudTrail event selector to log specific S3 events, you can use the `CloudTrailProps` configuration object
-
-For example - this logs all ReadWriteEvents for the `magic-bucket` bucket:
+For using CloudTrail event selector to log specific S3 events,
+you can use the `CloudTrailProps` configuration object.
```

Comment:
I'm a little confused because I don't see `CloudTrailProps` anywhere in the code example...